Reason: Prevent failure when applying Exherbo's default cmake parameters
Upstream: http://reviews.llvm.org/D18116

From 17bf35c69c49109c51e01a9a6439f9dcb28c81a2 Mon Sep 17 00:00:00 2001
From: Bernd Steinhauser <berniyh@exherbo.org>
Date: Fri, 11 Mar 2016 19:30:27 +0100
Subject: [PATCH] Allow CMAKE_BUILD_TYPE=None.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2c07e84..e059f67 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -180,7 +180,7 @@ endif()
 string(TOUPPER "${CMAKE_BUILD_TYPE}" uppercase_CMAKE_BUILD_TYPE)
 
 if (CMAKE_BUILD_TYPE AND
-    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
+    NOT uppercase_CMAKE_BUILD_TYPE MATCHES "^(DEBUG|NONE|RELEASE|RELWITHDEBINFO|MINSIZEREL)$")
   message(FATAL_ERROR "Invalid value for CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 endif()
 
-- 
2.7.2

